tutorials/backup-postgresql-pgbackrest-s3/index.mdx

```diff
@@ -0,0 +1,256 @@
+---
+meta:
+  title: Backup PostgreSQL databases with pgBackRest to Object Storage
+  description: How to install pgBackRest on Scaleway Instance for backing up PostgreSQL databases to Object Storage
+content:
+  h1: Backup PostgreSQL databases with pgBackRest to Object Storage
+  paragraph: This page shows how to backup PostgreSQL databases with pgBackRest to Object Storage
+tags: compute networking database sql postgresql pgbackrest backup buckets Object-Storage s3
+steps:
+  - step: Installing pgBackRest
+    type: HowToStep
+    url: https://www.scaleway.com/en/docs/tutorials/setup-pgbackrest/#-Installing-pgBackRest
+  - step: Setting up pgBackRest
+    type: HowToStep
+    url: https://www.scaleway.com/en/docs/tutorials/setup-pgbackrest/#-Setting-up-pgBackRest
+  - step: Setting up PostgreSQL
+    type: HowToStep
+    url: https://www.scaleway.com/en/docs/tutorials/setup-pgbackrest/#-Setting-up-PostgreSQL
+  - step: Performing backups
+    type: HowToStep
+    url: https://www.scaleway.com/en/docs/tutorials/setup-pgbackrest/#-Performing-backups
+  - step: Restoring from backups
+    type: HowToStep
+    url: https://www.scaleway.com/en/docs/tutorials/setup-pgbackrest/#-Restoring-from-backups
+  - step: Automatizing backups
+    type: HowToStep
+    url: https://www.scaleway.com/en/docs/tutorials/setup-pgbackrest/#-Automatizing-backups
+ - step: Deleting old backups manually
+    type: HowToStep
+    url: https://www.scaleway.com/en/docs/tutorials/setup-pgbackrest/#-Deleting-old-backups-manually
+categories:
+  - tutorials
+---
+
+## pgBackRest Overview
+pgBackRest aims to be a reliable, easy-to-use backup and restore solution that can seamlessly scale up to the largest databases and workloads by utilizing algorithms that are optimized for database-specific requirements.
+
+pgBackRest can backup in three ways:
+- Full backup: It is a backup of every file under the database directory ($PGDATA). A full backup of a PostgreSQL server is required first. It is the only backup that pgBackRest can load by itself.
+- Differential backup: It only retrieves files that have changed since the last full backup. Both the full backup and the differential backup must be intact.
+- Incremental backup: It retrieves files that have changed since the last backup. The last backup can be a full backup or a differential backup.
+
+<Message type="requirement">
+
+- You have an account and are logged into the Scaleway console.
+- You have created instances running a PostgreSQL database on Ubuntu/Debian or RHEL/Rocky Linux. -> https://www.scaleway.com/en/docs/tutorials/back-up-postgresql-barman/
```

Comment:
```suggestion
- You have created Instances running a [PostgreSQL database](https://www.scaleway.com/en/docs/tutorials/install-postgresql/) on Ubuntu/Debian or RHEL/Rocky Linux.
```

Reply:
I'm not sure why there is the link to the barman tutorial - please feel free to reinstate it if there's a reason I'm not understanding! Also, if you could specify how many Instances are required that would be great.

Reply:
Oh sorry. I referenced the barman tutorial for styling the tutorial. I forgot to delete that part. My bad. :(